pyaedt/hfss3dlayout.py

```diff
@@ -1254,8 +1257,9 @@ def import_gds(self, gds_path, aedb_path=None, control_file=None, set_as_active=
         aedb_path : str, optional
             Full path to the AEDB file.
         control_file : str, optional
-            Path to the XML file with the stackup information. The default is ``None``, in
+            Path to the XML or tech file with the stackup information. The default is ``None``, in
             which case the stackup is not edited.
+            If `tech` file is provided and layer name starts with `"v"` then is mapped as via layer.
```

Comment:
```suggestion
            If a TECH file is provided and the layer name starts with ``"v"``, the layer is mapped as a via layer.
```
It's unclear here whether the pronoun "it" refers to the TECH file or the layer. I assumed layer, but I'd like you to check.

Reply:
Fixed. Tks